### PR TITLE
fix scalafmt plugin

### PR DESCRIPTION
**Pull request by jrudolph**
Was declared in wrong file

### CODE REVIEW

**Comment by jrudolph** [comment]
There's nothing to format?!?

File: project/plugins.sbt [hunk 1]
```diff
@@ -2,3 +2,4 @@
 // remember to also update build.sbt
 addSbtPlugin("com.lightbend.paradox" % "sbt-paradox-theme" % "0.10.3")
 addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.5.11")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.6")
```
Comment (mdedetrich) [inline project/plugins.sbt:5]: @jrudolph Newline is missing, scalafmt adds new lines

Comment (jrudolph) [reply project/plugins.sbt]: :+1: done

**Review by mdedetrich — APPROVED** [review]
lgtm

**Comment by mdedetrich** [comment]
Going ahead to merge this.

**Pull request merged**